articles/case_studies/fec-javascript-testing.md

```diff
@@ -0,0 +1,70 @@
+# Testing and refactoring JavaScript with FEC
+
+Until August of this year, the JavaScript for the OpenFEC project had no unit tests. This wasn't because the development team wasn't interested in testing--our backend code was already thoroughly tested. Instead, our JavaScript code just wasn't pleasant to test. Here's an example of the coding style that prevailed not long ago:
+
+```javascript
+$(document).ready(function() {
+  var $widget = $('#widget');
+  $widget.on('click', function() {
+    $.getJSON($widget.data('url')).done(function(response) {
+      $widget.find('.content').text(response.content);
+    }).fail(function(error) {
+      alert('It broke!');
+    });
+  });
+});
+```
+
+Because this is a toy example, it's not too hard to tell what this code does:
```

Comment:
"Because this is a toy example, it's not too hard to tell what this code does:" -> "It's not too hard to tell what this code does:"